feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-trigger/product-image-zoom-trigger.component.ts

```diff
@@ -26,6 +28,8 @@ export class ProductImageZoomTriggerComponent implements OnDestroy {
   iconType = ICON_TYPE;
   protected subscriptions = new Subscription();
 
+  @ViewChild('expandButton') expandButton: ElementRef;
```

Comment:
Lets just have a comment here about why we need to reference this button (ie. to focus).